tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -694,6 +837,24 @@ TEST(RNTuple, TClassEBO)
    }
 }
 
+TEST(RNTuple, IOConstructor)
+{
+   FileRaii fileGuard("test_ntuple_ioconstructor.ntuple");
+
+   auto model = RNTupleModel::Create();
+   auto fldObj = RFieldBase::Create("obj", "IOConstructor").Unwrap();
```

Comment:
Should we also add a test with `std::unique_ptr<IOConstructor>` ?

Reply:
[Done|https://github.com/root-project/root/pull/12664/commits/364f89e3d59a2a7f00a356b147540a57c022d373]